shared/constants/keyring.ts

```diff
@@ -24,3 +29,40 @@ export const KeyringType = {
   ...SnapKeyringType,
   ///: END:ONLY_INCLUDE_IF
 };
+
+// TODO: This kind of logic should be inside the `KeyringController` (using `KeyringSelector` query, or make `addNewKeyring` returns it keyring ID alongside
+export function findKeyringId(
```

Comment:
We shouldn't have a function in the `shared/constants` folder IMO. We should probably move it to `shared/lib/keyring.ts`.